.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: ci-tests
+
+on:
+  pull_request:
+
```

Comment:
```suggestion
  pull_request:
  workflow_dispatch:
```

Reply:
Soz, I missed this and already pushed the change ... speedy suggestion though 😮‍💨